### ISSUE TITLE
Report not showing properly on centos7

### ISSUE THREAD

**Issue by ezequielzacca** [jsreport/jsreport-phantom-pdf#24]
I have a report wich works perfectly on windows, but when i create it on my test environment using centos7, it shows black squares instead of normal characters
on centos
![image](https://user-images.githubusercontent.com/1986425/32728166-0e820f32-c85e-11e7-91d6-b35892f2023f.png)

same on windows
![image](https://user-images.githubusercontent.com/1986425/32728249-544d58b4-c85e-11e7-87c1-4f599c5190d9.png)

**Comment by ezequielzacca** [comment]
now, after running sudo yum install freetype fontconfig on the server, it changed, but still wrong
![image](https://user-images.githubusercontent.com/1986425/32728489-5393c42a-c85f-11e7-94b1-d9fea2e2f768.png)

**Comment by pofider** [comment]
Please try [these steps](https://github.com/jsreport/docs/blob/master/installation/centos.md#fixing-pdf-fonts).

**Comment by ezequielzacca** [comment]
i changed the font on the css of the report to another font, and i installed such font on centos by copying it onto '/usr/share/fonts' folder, and after that it worked properly, the only issue i found (wich i assume is not a jsreport issue but probably a windows/linux related issue) is that the size of the fonts are different on windows and linux

**Comment by pofider** [comment]
Yes, this is [documented phantomjs issue](https://jsreport.net/learn/phantom-pdf#different-sizes-on-windows-vs-unix).
